wiggum_app/static/js/rankchart.js

```diff
@@ -210,9 +210,10 @@ function DrawRankChart(data) {
  * @param colLabels - feat2 used for columns labels.
  * @param category - groupfeat and subgroup.
  * @param trend_type - trend type.
+ * @param detail_view_type - detail view type.
  * @returns matrix - containing information for cells.
  */
-var UpdateRankTrendMatrixFormat = function(matrix, rowLabels, colLabels, groupInfo, trend_type) {
+var UpdateRankTrendMatrixFormat = function(matrix, rowLabels, colLabels, groupInfo, trend_type, detail_view_type) {
```

Comment:
Do you still use the trend type? or does detail view type replace trend_type?

Reply:
I use detail view type for interacting with detail view, but keep trend_type for interactive with result table.